examples/src/main/kotlin/dashj/org/platform/RegisteredNamesWithBalance.kt

```
@@ -0,0 +1,85 @@
+                    }
+                } catch (e: Exception) {
+                    println("\nError retrieving results (startAt =  $lastItem)")
+                    println(e.message)
+                    return
+                }
+            } while (requests >= 0 && documents!!.size == limit)
+
+            val nameBalanceMap = hashMapOf<String, Long>()
+            allDocuments.forEachIndexed { i, document ->
+                val nameDocument = DomainDocument(document)
+                val balance = try { platform.client.getIdentityBalance(nameDocument.dashUniqueIdentityId!!) } catch (e: Exception) { - 2 }
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)